src/components/Loader/Error.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+interface ErrorMessage {
+  error: string;
+}
+export const Error: React.FC<ErrorMessage> = ({ error }) => {
+  return (
```

Comment:
return jsx without word `return` (if linter allows)
check in all other components